src/testMod/resources/data/advancedperipheralstest/computer/tests/craftos_test.sends_basic_rednet_messages.echo.lua

```diff
@@ -0,0 +1,8 @@
+os.getComputerID = function() return 1 end
```

Comment:
I don't think we need to test CC:T stuffs in AP.

Reply:
Yeah these are just for testing. I used them to see if everything works fine in the testing environment